apps/explorer/src/components/ui/ObjectVideoImage.tsx

```diff
@@ -84,25 +95,23 @@ export function ObjectVideoImage({
                 onClose={close}
                 title={title}
                 subtitle={subtitle}
-                src={src}
-                video={video}
+                src={src || ''}
                 alt={title}
             />
-            <div className={imageStyles({ variant, disablePreview })}>
-                <Image
-                    aspect={aspect}
-                    rounded={rounded}
-                    onClick={openPreview}
-                    alt={title}
+            <div
+                className={clsx(
+                    imageStyles({ variant, disablePreview, rounded }),
+                    rounded && 'overflow-hidden',
+                )}
```

Comment:
to keep the bg :pray: 
![image](https://github.com/user-attachments/assets/2bdc9493-7a0a-4e2e-9249-850e0cdaa9f1)
instead of:
![image](https://github.com/user-attachments/assets/7b28cda9-2647-4abd-a5bb-339c5c2bd9b8)

```suggestion
                className={clsx(
                    'bg-neutral-96 dark:bg-neutral-10',
                    imageStyles({ variant, disablePreview, rounded }),
                    rounded && 'overflow-hidden',
                )}
```